src/periphery/Rewarder.sol

```diff
@@ -150,24 +157,24 @@ contract Rewarder is IRewarder {
 
     /**
      * @notice Withdraw SPELL and harvest the rewards
-     * @param from user for which amount is withdrawn 
+     * @param from user for which amount is withdrawn
      * @param _amount The amount of SPELL to withdraw
      */
-    function withdraw(address from, uint256 _amount) external onlyCauldron {
+    function withdraw(address from, uint256 _amount) external override onlyCauldron {
         UserInfo memory user = userInfo[from];
```

Comment:
iirc if you use `UserInfo storage user = userInfo[from];` this is going to write to storage directly.

Reply:
yes, i often prefer handling storage writes more explicitly.